Makefile

```diff
@@ -76,6 +76,8 @@ test-v: clean fmt REBUILD
 	go test -race -v -short -tags='debug testing' -timeout=15s $(pkgs)
 test-long: clean fmt REBUILD
 	go test -v -race -tags='testing debug' -timeout=180s $(pkgs)
+bench: clean fmt REBUILD
+	go test -tags='testing' -timeout=600s -run=XXX -bench=. $(pkgs)
```

Comment:
10 minute timeout is too much, for now it should probably be between 60s and 120s